pages.pt_BR/common/gpg.md

```diff
@@ -0,0 +1,32 @@
+# gpg
+
+> GNU Privacy Guard.
+> Mais informações: <https://gnupg.org>.
+
+- Assina doc.txt, sem criptografá-lo (cria um arquivo de saída doc.txt.asc):
+
+`gpg --clearsign {{doc.txt}}`
+
+- Criptografa doc.txt para alice@example.com (cria um arquivo de saída doc.txt.gpg):
+
+`gpg --encrypt --recipient {{alice@example.com}} {{doc.txt}}`
+
+- Criptografa doc.txt com uma senha simétrica somente (cria um arquivo de sadída doc.txt.gpg):
```

Comment:
@caduvieira: what do you think about that?

```suggestion
- Criptografa doc.txt apenas com uma senha simétrica (cria um arquivo de sadída doc.txt.gpg):
```

Reply:
I think we can use your suggestion.

Reply:
cool :)